user.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
```

Comment:
I strongly suggest you to stop adding many meaningless divs. Go with keeping only the essential divs and not nesting many divs one inside the other.